contracts/protocol/libraries/position/LiquidityBitmap.sol

```diff
@@ -95,6 +96,16 @@ library LiquidityBitmap {
                         }
                     }
                 }
+                if (i == 0 && _self[0] != 0) {
```

Comment:
find liquidity at word 0. we can't do this in for loop cause at the end of the loop, variable `i` will be minus by 1 and cause an overflow